README.md

```diff
@@ -3,33 +3,76 @@
 __URLs__
 
 * Name all of the parts of the url that you can remember.  In your own words describe what they do.
+```
+Protocol: the connection in which you access the webpage?
+The port: typically 80. dont remember much else
+The path: the file path that of the website.
+the domain: the URL which represents an IP address
+the query: What is being searched for
+anchor tag: link to specific part of page
+
+```
 * Name the pieces of the following urls:
 	* `https://www.google.com/`
+	```
+	Protocol - domain -port
```

Comment:
👍 port isn't actually included, so it just uses the default port for the protocol